src/main/webapp/resources/js/components/samples/components/SampleInfo.tsx

```diff
@@ -67,6 +72,10 @@ export function SampleInfo() {
       });
   };
 
+  const ontologyProps = {
```

Comment:
Why are you making this props objects?

Reply:
Same comment as above. Typescript error and creating a const fixes it

Reply:
Removed in 409e17b4867a1ce1d38b985621e66a46f4db2f48